docs/setup-cluster/security/tls.rst

```diff
@@ -75,3 +75,247 @@ untrusted certificate and display a hash of the certificate. You may wish to con
+   provider, this may take some time, from a few seconds to multiple minutes. You can
+   check if it has finished deploying with the aid of online tools, such as the Google
+   Admin Toolbox: https://toolbox.googleapps.com/apps/dig/#TXT/_acme-challenge.<domain>.
+   Look for one or more bolded line(s) below the line ';ANSWER'. It should show the
+   value(s) you've just added.
+
+   Press Enter to Continue
+
+.. caution::
+
+   DO NOT PRESS ENTER BEFORE SETTING UP THE DNS RECORD
+
+Set Up the DNS Record Before Pressing ``ENTER``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+DNS TXT Record example
+
++---------------------------------+-------------+-----+-------------------------------------------------+
+| FQDN                            | RECORD TYPE | TTL | Value                                           |
++=================================+=============+=====+=================================================+
+| _acme-challenge.<domain>.       | TXT         | 900 | <XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX  |
+|                                 |             |     | domain token>                                   |
++---------------------------------+-------------+-----+-------------------------------------------------+
+
+Ensure the ``_acme-challenge.<domain>.`` DNS record has been propagated using one of the following:
+
+-  ``https://toolbox.googleapps.com/apps/dig/#TXT/_acme-challenge.<domain>.``, or
+-  ``nslookup -type=TXT _acme-challenge.<domain>.``
+
+.. note::
+
+   You may need to install bind-utils to run ``nslookup`` using the following command:
+
+   .. code:: bash
+
+      yum install bind-utils
+
+Press ``ENTER``
+^^^^^^^^^^^^^^^
+
+Once you have set up the DNS record, press **Enter**.
+
+Certbot responds, letting you know it has received the certificate. Certbot provides the certificate
+location, key and the certificate expiration date.
+
+.. Important::
+
+   To renew the certificate repeat the certificate creation steps.
+
+Determined Master TLS Configuration
+===================================
+
+This section describes how to use the TLS certs provided by the Let’s Encrypt service.
+
+First, stop the Determined Master with the following command:
+
+.. code:: bash
+
+   systemctl stop determined-master
+
+Then, change the security branch of the ``master.yaml`` by adding the following configuration:
+
+.. code:: bash
+
+   security:
+
+   tls:
+
+      cert: /etc/letsencrypt/live/<domain>/fullchain.pem
+
+      key: /etc/letsencrypt/live/<domain>/privkey.pem
+
+Eventually, change the master port:
+
+.. code:: bash
+
+   # master port
+   port: 443
+
+.. Important::
+
+   You'll need to configure the agents to reach this port.
+
+Finally, start the Determined Master with the following command:
+
+.. code:: bash
+
+   systemctl start determined-master
+
+After a certificate renewal, you must restart the Determined Master using the following command:
+
+.. code:: bash
+
+   systemctl restart determined-master
```

Comment:
Same thing as above about assuming the installation method.

And, as always, it is "master", not "Master".

Reply:
resolved